### PR TITLE
Lighthouse validator import

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

File: controls/roles/validator-import/molecule/default/stereum-pwd [hunk 1]
```diff
@@ -0,0 +1 @@
+Sk,~;g:~91B}l4@kO=>&`?<A7n6+3\0zeI3!uEsf/e$*|#9wjSTz:d<8v3gAQ*ou
```
Comment (stefa2k) [inline controls/roles/validator-import/molecule/default/stereum-pwd:1]: please delete this file

File: controls/roles/validator-import/molecule/default/verify.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    register: stereum_services_dir
+  - debug:
+      msg: "{{ stereum_services_dir }}"
+  - shell: docker ps
+    register: stereum_docker_ps
+  - debug:
+      msg: "{{ stereum_docker_ps }}"
+  - assert:
+      that:
+      - stereum_docker_ps.stdout.find("stereum/lighthouse") != -1
```
Comment (stefa2k) [inline controls/roles/validator-import/molecule/default/verify.yml:16]: can we check somehow that the validator is loading the keys correctly? or at least that it connected to the beacon node?

File: controls/roles/validator-import/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+              }
+              exp_continue
+            }
+        } 
+      owner: "{{ stereum_service_configuration.user }}"
+      group: "2000"     
+      mode: 0700
+    become: yes   
+
+  - name: Start beacon service
```
Comment (stefa2k) [inline controls/roles/validator-import/tasks/main.yml:95]: can you put this into `converge.yaml` please? when we run this role the beacon and validator should already run

you can make use of the "manage-service" role and call with parameters:
https://github.com/stereum-dev/ethereum-node/blob/main/controls/roles/manage-service/molecule/start-multiple/converge.yml#L10

File: controls/roles/validator-import/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        - name: stereum
+      # working_dir: "{{ stereum_service_configuration.working_dir }}" # doesn't work at all
+      volumes: "{{ stereum_service_configuration.volumes }}"
+      log_driver: "json-file"
+      log_options:
+        max-file: "10"
+        max-size: "100m"
+    become: yes               
+
+  - name: Start validator service
```
Comment (stefa2k) [inline controls/roles/validator-import/tasks/main.yml:128]: same as beacon, please run in converge.yaml

File: controls/roles/validator-import/tasks/write-configuration.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (stefa2k) [inline controls/roles/validator-import/tasks/write-configuration.yml:1]: why do we need this here?

File: controls/roles/validator-import/templates/service.yaml [hunk 6]
```diff
@@ -0,0 +1 @@
+{{ stereum.manage_service.configuration | to_nice_yaml(indent=2) }}
```
Comment (stefa2k) [inline controls/roles/validator-import/templates/service.yaml:1]: I think that's copy&paste, no purpose right?

File: controls/roles/validator-import/templates/service.yml [hunk 7]
```diff
@@ -0,0 +1 @@
+{{ stereum.manage_service.configuration | to_nice_yaml(indent=2) }}
```
Comment (stefa2k) [inline controls/roles/validator-import/templates/service.yml:1]: I think that's copy&paste, no purpose right?

File: controls/roles/validator-import/vars/main.yml [hunk 8]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (stefa2k) [inline controls/roles/validator-import/vars/main.yml:1]: it's empty, please remove this file

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: controls/roles/validator-import/README.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
+Role Name
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/validator-import/README.md:1]: got it, README.md file is removed

File: controls/roles/validator-import/molecule/default/INSTALL.rst [hunk 10]
```diff
@@ -0,0 +1,16 @@
+***************************************
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/validator-import/molecule/default/INSTALL.rst:1]: removed

File: controls/roles/validator-import/molecule/default/molecule.yml [hunk 11]
```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: hetznercloud
+platforms:
+  - name: "validator--import--ubuntu-20.04"
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/validator-import/molecule/default/molecule.yml:7]: good to know, thanks. it is renamed

File: controls/roles/validator-import/molecule/default/molecule.yml [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+    hostname: ubuntu
+    server_type: cpx21
+    image: ubuntu-20.04
+  - name: "role-manage-service-centos-8"
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/validator-import/molecule/default/molecule.yml:11]: good to know, thanks. it is renamed

File: controls/roles/validator-import/molecule/default/stereum-pwd (hunk 1 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/molecule/default/stereum-pwd:1]: deleted

File: controls/roles/validator-import/templates/service.yaml (hunk 6 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/templates/service.yaml:1]: yes, was just copy&paste and I'll delete it

File: controls/roles/validator-import/templates/service.yml (hunk 7 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/templates/service.yml:1]: yes, was just copy&paste and I'll delete it

File: controls/roles/validator-import/tasks/write-configuration.yml (hunk 5 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/tasks/write-configuration.yml:1]: it was also copy & paste

Comment (gbayasgalan) [reply controls/roles/validator-import/tasks/write-configuration.yml:1]: removed

File: controls/roles/validator-import/molecule/default/molecule.yml [hunk 13]
```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: hetznercloud
+platforms:
+  - name: "validator--import--default--ubuntu-20.04"
```
Comment (stefa2k) [inline controls/roles/validator-import/molecule/default/molecule.yml:7]: `validator-import--default--ubuntu-20.04` the separator is always two minus like this: `--`

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: controls/roles/validator-import/molecule/default/molecule.yml (hunk 13 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/molecule/default/molecule.yml]: okey, got it

File: controls/roles/validator-import/molecule/default/verify.yml (hunk 2 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/molecule/default/verify.yml:16]: done!

File: controls/roles/validator-import/tasks/main.yml (hunk 3 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/tasks/main.yml]: done!

File: controls/roles/validator-import/tasks/main.yml (hunk 4 above)
Comment (gbayasgalan) [reply controls/roles/validator-import/tasks/main.yml]: done!

**Review by stefa2k — COMMENTED** [review]
lgtm

**Pull request merged**